recipes/mqc/run_test.pl

```diff
@@ -0,0 +1,11 @@
+# Test file for testing Perl modules installation
+$|=1;
```

Comment:
And how can you make sure that he uses the perl 'use' tests for the perl modules instead of the python 'import' test, which he needs to use for the python modules?